website/src/app/pages/data-distributions/data-distributions.component.html

```diff
@@ -1,2 +1,6 @@
-<agc-table-data-selector></agc-table-data-selector>
+<agc-table-data-selector
+  [datasets]="$any(data.datasets$ | async)"
```

Comment:
What is `$any`?

Reply:
it opts that field out of type checking. without it I get the error:  Type 'Dataset[] | null' is not assignable to type 'Dataset[]'.  Do you know how to resolve without using it? This was the solution I had found online when looking into it but there may be a better way

Reply:
Remove the `!` in https://github.com/cns-iu/a2agc-dataset/blob/pbi-80-data-distribution/website/src/app/shared/components/table-data-selector/table-data-selector.component.ts#L14 initialize to empty array and do null checking where appropriate.